adapters/connatix/connatix.go

```diff
@@ -164,9 +165,38 @@ func splitRequests(imps []openrtb2.Imp, request *openrtb2.BidRequest, uri string
 			return nil, errs
 		}
 
+		endpoint, err := url.Parse(uri)
+		if err != nil {
+			errs = append(errs, err)
+			fmt.Println("cnx_errors: ", errs)
```

Comment:
do we need to print in prod ? Seems like a line only needed for debugging locally. Might be best to remove

Reply:
fixed